### PR TITLE
Made Standard Readme compliant

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/community/issues/124.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
 # apps
-Coordinating writing apps on top of ipfs, and their concerns.
 
-This is a discussion repo. That means that all of the work gets done in the [issues](https://github.com/ipfs/apps/issues). Go check them out! 
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
```
Comment (davidar) [inline README.md]: Personally I'd prefer this to link to https://chat.ipfs.io/#/room/#ipfs:matrix.org

Comment (RichardLitt) [inline README.md]: Huh, haven't seen this before. When was this set up? Seems like a lot of extra stuff more than just the IRC room.

Comment (davidar) [inline README.md]: @RichardLitt davidar/vector-web#2

Comment (RichardLitt) [inline README.md]: Huh. Interesting! @jbenet @diasdavid should we use this, instead?

Comment (diasdavid) [inline README.md]: 👍

Comment (RichardLitt) [inline README.md]: It doesn't seem to work. I am trying it, and it is not sending the message. Perhaps we should stick with freenode for now?

Comment (davidar) [inline README.md]: @RichardLitt works for me. Did you click the "join" link first?

Comment (RichardLitt) [inline README.md]: I did. I typed messages, but didn't see them coming up in IRCCloud. :?

Comment (davidar) [inline README.md]: thanks for testing @RichardLitt, I think I know what the problem is now, will try to get it fixed asap

Comment (davidar) [inline README.md]: I'm told this problem should be fixed sometime today.

Comment (davidar) [inline README.md]: @RichardLitt can you check that https://ipfs.io/ipfs/QmcSJjw7wQ8r8uUKBHb8n2RryXsKv49MS31xFwowxkgJ53/#/room/#ipfs:matrix.org works?

@lgierth if so, can you update the chat.ipfs.io dnslink to the above hash?

Comment (davidar) [inline README.md]: @RichardLitt https://chat.ipfs.io/#/room/#ipfs:matrix.org should work now, please update the link if you're happy with it :)

**Comment by JesseWeinstein** [comment]
LGTM, aside from the discussion about linking to Matrix vs Freenode webchat.

**Comment by RichardLitt** [comment]
@davidar Undecided at the moment about chat.ipfs.io -- I'm going to merge this, for now, and will change all of the links from freenode's webchat later.

**Comment by davidar** [comment]
> will change all of the links from freenode's webchat later.

@RichardLitt ok, I'll hold you to that ;)

**Comment by jbenet** [comment]
This was brought up to me recently I think. The concerns were that the
chat.ipfs.io UX loses people. I think we can get vector UX much closer to
being purely focused on the channel.

**Pull request merged**